example/pages/index.js

```diff
@@ -3,7 +3,7 @@ import path from 'path'
 import fs from 'fs'
 import { useRemoteRefresh } from 'next-remote-refresh/hook'
 
-export default function Index({ name, version, description }) {
+export default function Index ({ name, version, description }) {
```

Comment:
Just a personal preference, but can we keep no spaces between identifiers and params 😇

Reply:
honestly I also detest this convention so I'd be very happy to update this rule 🚀